srsly/__init__.py

```diff
@@ -1,4 +1,10 @@
-from ._json_api import read_json, read_gzip_json, write_json, write_gzip_json
+from ._json_api import (
+    read_json,
+    read_gzip_json,
+    write_json,
+    write_gzip_json,
+    write_gzip_jsonl,
+)
```

Comment:
I meant `read_gzip_jsonl`, and our usual style is to modify the imports manually to avoid the multi-line formatting.

```suggestion
from ._json_api import read_json, read_gzip_json, write_json, write_gzip_json
from ._json_api import read_gzip_jsonl, write_gzip_jsonl
```